packages/cloud_firestore/lib/src/collection_reference.dart

```diff
@@ -22,6 +25,10 @@ class CollectionReference extends Query {
         _firestore, (new List<String>.from(_pathComponents)..removeLast()));
   }
 
+  /// A string containing the slash-separated path to this  CollectionReference
```

Comment:
Remove extra space.
No need to say this is a string. `/// Slash-separated path to...` should do fine.